source/_components/gocr_image_processing.markdown

```diff
@@ -0,0 +1,150 @@
+
```

Comment:
Missing the standard doc settings https://developers.home-assistant.io/docs/en/documentation_create_page.html